collect_app/src/main/java/org/odk/collect/android/forms/DatabaseMediaFileRepository.java

```diff
@@ -12,7 +12,10 @@
     @Override
     public List<File> getAll(String jrFormID, String formVersion) {
         String formMediaPath = new FormsDao().getFormMediaPath(jrFormID, formVersion);
-        File[] files = new File(formMediaPath).listFiles();
+        File[] files = new File[0];
```

Comment:
I think we could avoid the array initialization here with an `if-else` on the `formMediaPath != null` and just returning `emptyList` in the `else` case. The `new File[0]` going into the `asList` confused me.

Reply:
Done.